aptos-move/framework/aptos-framework/sources/transaction_context.spec.move

```diff
@@ -2,7 +2,10 @@ spec aptos_framework::transaction_context {
     spec get_script_hash(): vector<u8> {
         pragma opaque;
         aborts_if false;
+        // property 4: Fetching the script hash of the current entry function should never fail
+        // and should return a vector with 32 bytes if the transaction payload is a script, otherwise an empty vector.
         ensures result == spec_get_script_hash();
```

Comment:
Can you put [abstract] for these `ensures` conditions?

Reply:
sure